neoexchange/core/templates/core/data_summary.html

```diff
@@ -80,6 +156,25 @@ <h1 style="display:inline"> LCO Light Curves </h1> <a href="{% url 'spec_data_su
             </tbody>
         </table>
     </div>
+    {% if request.user.is_authenticated and perms.core.add_body %}
```

Comment:
Wondering whether to add a new and different `perms` for this.. Downside is it's harder to setup new users, upside is finer grained permissions to allow updates of a Body's parameters rather than wholesale new Body's. Probably don't have enough relevant users for the distinction to matter at this stage...

Reply:
I'll leave this up to you :D